spec/plugin_spec.rb

```diff
@@ -254,35 +254,25 @@ def assert_user_following_state(user, channel, following:)
 
     describe "when a user is created" do
       fab!(:category) { Fabricate(:category) }
-      fab!(:user) { Fabricate(:user) }
+      let(:user) { Fabricate(:user, last_seen_at: nil, first_seen_at: nil) }
```

Comment:
do we need let now?

Reply:
Yes. Re-using the user with `fab!` makes the first test flaky.